src/js/pages/jobs/JobConfiguration.js

```diff
@@ -27,36 +26,36 @@ class JobConfiguration extends React.Component {
         <ConfigurationMapHeading>General</ConfigurationMapHeading>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>ID</ConfigurationMapLabel>
-          <ConfigurationMapValue>{job.getId()}</ConfigurationMapValue>
+          <ConfigurationMapValue>{job.id}</ConfigurationMapValue>
         </ConfigurationMapRow>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>Description</ConfigurationMapLabel>
-          <ConfigurationMapValue>{job.getDescription()}</ConfigurationMapValue>
+          <ConfigurationMapValue>{job.description}</ConfigurationMapValue>
         </ConfigurationMapRow>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>CPUs</ConfigurationMapLabel>
-          <ConfigurationMapValue>{job.getCpus()}</ConfigurationMapValue>
+          <ConfigurationMapValue>{job.cpus}</ConfigurationMapValue>
         </ConfigurationMapRow>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>Memory (MiB)</ConfigurationMapLabel>
-          <ConfigurationMapValue>{job.getMem()}</ConfigurationMapValue>
+          <ConfigurationMapValue>{job.mem}</ConfigurationMapValue>
         </ConfigurationMapRow>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>Disk Space (Mib)</ConfigurationMapLabel>
-          <ConfigurationMapValue>{job.getDisk()}</ConfigurationMapValue>
+          <ConfigurationMapValue>{job.disk}</ConfigurationMapValue>
         </ConfigurationMapRow>
         <ConfigurationMapRow>
           <ConfigurationMapLabel>Command</ConfigurationMapLabel>
           <ConfigurationMapValue>
-            <pre className="flush transparent wrap">{job.getCommand()}</pre>
+            <pre className="flush transparent wrap">{job.command}</pre>
           </ConfigurationMapValue>
         </ConfigurationMapRow>
       </ConfigurationMapSection>
     );
   }
 
   getScheduleSection(job) {
-    const [schedule] = job.getSchedules();
+    const [schedule] = job.schedules.nodes;
```

Comment:
Why the thing inside the schedules is called nodes? More like a curiosity since I don't see the relationship.

Reply:
Thanks for the clarification. Is that possible we do not leak this abstraction? Like the mediator process it in a way that I can call "jobschedules". I find it as weird as if I had to do `user.joins.addresses` on a traditional database system. The UI components should not care about nodes.

Reply:
AFAIK: no